### PR TITLE
created a table for raidInstance

### LINKED ISSUE #13: Database - Design Raid Instance Table
Raid instance table should associate with a `raid`. It should contain the information specific to an individual run of the raid (like users, run time)

### PR DESCRIPTION
**Pull request by Dustin-Davis**

### CODE REVIEW

**Comment by WhiskeyJack96** [comment]
Same as the other two, but also game id isnt on this table so you probably dont want a foriegn key on it for game id

File: server/sql/schema.sql [hunk 1]
```diff
@@ -9,6 +9,17 @@ CREATE TABLE "game" (
     "gameName" TEXT NOT NULL,
     "createdAt" TIMESTAMPTZ NOT NULL default now(),
     CONSTRAINT "games_pk" PRIMARY KEY ("gameId")
-) WITH (
-  OIDS=FALSE
 );
+
+CREATE TABLE "raidInstance" (
```
Comment (WhiskeyJack96) [inline server/sql/schema.sql]:
```suggestion
CREATE TABLE "raid_instance" (
```

File: server/sql/schema.sql [hunk 2]
```diff
@@ -9,6 +9,17 @@ CREATE TABLE "game" (
...
+    "raid_Id" INTEGER NOT NULL,
+    "user_Id" INTEGER NOT NULL,
+    "run_Time" TIME NOT NULL,
+    "raid_Leader" TEXT NOT NULL,
+    "selected_Roles" TEXT NOT NULL,
+    "created_At" TIMESTAMPTZ NOT NULL default now(),
+    CONSTRAINT "raidInstance_pk" PRIMARY KEY ("raid_Instance_Id")
+);
+
+ALTER TABLE "raidInstance" ADD CONSTRAINT "raidInstance_fk0" FOREIGN KEY ("raid_Id") REFERENCES "raid"("raid_Id")
```
Comment (WhiskeyJack96) [inline server/sql/schema.sql]:
```suggestion
ALTER TABLE "raid_instance" ADD CONSTRAINT "raid_instance_fk0" FOREIGN KEY ("raid_Id") REFERENCES "raid"("raid_Id")
```

File: server/sql/schema.sql [hunk 3]
```diff
@@ -9,6 +9,17 @@ CREATE TABLE "game" (
...
+
+CREATE TABLE "raidInstance" (
+    "raid_Instance_Id" INTEGER GENERATED ALWAYS AS IDENTITY NOT NULL,
+    "raid_Id" INTEGER NOT NULL,
+    "user_Id" INTEGER NOT NULL,
+    "run_Time" TIME NOT NULL,
+    "raid_Leader" TEXT NOT NULL,
+    "selected_Roles" TEXT NOT NULL,
+    "created_At" TIMESTAMPTZ NOT NULL default now(),
+    CONSTRAINT "raidInstance_pk" PRIMARY KEY ("raid_Instance_Id")
```
Comment (WhiskeyJack96) [inline server/sql/schema.sql]:
```suggestion
    CONSTRAINT "raid_instance_pk" PRIMARY KEY ("raid_instance_Id")
```

File: server/sql/schema.sql [hunk 4]
```diff
@@ -9,6 +9,17 @@ CREATE TABLE "game" (
...
 );
+
+CREATE TABLE "raidInstance" (
+    "raid_Instance_Id" INTEGER GENERATED ALWAYS AS IDENTITY NOT NULL,
+    "raid_Id" INTEGER NOT NULL,
+    "user_Id" INTEGER NOT NULL,
+    "run_Time" TIME NOT NULL,
+    "raid_Leader" TEXT NOT NULL,
+    "selected_Roles" TEXT NOT NULL,
+    "created_At" TIMESTAMPTZ NOT NULL default now(),
```
Comment (WhiskeyJack96) [inline server/sql/schema.sql]:
```suggestion
    "raid_instance_id" INTEGER GENERATED ALWAYS AS IDENTITY NOT NULL,
    "raid_id" INTEGER NOT NULL,
    "user_id" INTEGER NOT NULL,
    "run_time" TIME NOT NULL,
    "raid_leader" TEXT NOT NULL,
    "selected_roles" TEXT NOT NULL,
    "created_at" TIMESTAMPTZ NOT NULL default now(),
```

File: server/sql/schema.sql [hunk 5]
```diff
@@ -9,6 +9,17 @@ CREATE TABLE "game" (
...
+    "raid_id" INTEGER NOT NULL,
+    "user_id" INTEGER NOT NULL,
+    "run_time" TIME NOT NULL,
+    "raid_leader" TEXT NOT NULL,
+    "selected_roles" TEXT NOT NULL,
+    "created_at" TIMESTAMPTZ NOT NULL default now(),
+    CONSTRAINT "raid_instance_pk" PRIMARY KEY ("raid_instance_Id")
+);
+
+ALTER TABLE "raid_instance" ADD CONSTRAINT "raid_instance_fk0" FOREIGN KEY ("raid_Id") REFERENCES "raid"("raid_Id")
```
Comment (WhiskeyJack96) [inline server/sql/schema.sql]:
```suggestion
    CONSTRAINT "raid_instance_pk" PRIMARY KEY ("raid_instance_id")
);

ALTER TABLE "raid_instance" ADD CONSTRAINT "raid_instance_fk0" FOREIGN KEY ("raid_id") REFERENCES "raid"("raid_id")
```

**Review by WhiskeyJack96 — APPROVED** [review]

**Review by Alex-Davis0 — APPROVED** [review]

**Pull request merged**